src/main.jsx

```diff
@@ -10,7 +10,7 @@ ReactDOM.render(
 
     <React.StrictMode>
       <App />
-    </React.StrictMode>,
```

Comment:
should we remove the comma

Reply:
Yup, it will remove the extra section that comes in page.